doc/ChangeLog

```diff
@@ -0,0 +1,93 @@
+
+The main purpose of this PR is to update the CCPP standard names to match the
+internal AMP agreed-upon names, as well as rename some of the physics routines
+to make them less ambigious and more accurate.  Along with these changes,
+significant code cleanup was performed along with some minor bug fixes,
+with the goal being to produce bit-for-bit results using the same repo
+version in both CAM and CAM-SIMA.
+
+This PR fixes the following NCAR/atmospheric_physics Github issues:
+
+#60 -> Required atmospheric physics updates
+#61 -> "calc_exner_run" routine should be given reference pressure argument
+#62 -> Need to update geopotential_t to use "generalized" virtual temperature
+#64 -> held_suarez_1994 SDF needs the geopotential_t scheme.
+
+Code reviewed by: cacraig, peverwhee, mwaxmonsky
+
+List all files eliminated:
+
+utilities/geopotential_t.F90 - Replaced with utilities/geopotential_temp.F90
+utilities/geopotential_t.meta - Replaced with utilities/geopotential_temp.meta
+
+List all files added and what they do:
+
+utilities/geopotential_temp.F90 - New geopotential calculation routine which uses
+                                  generalized virtual temperature (i.e. all
+                                  thermodynamically active species).
+utilities/geopotential_temp.meta - Metdata file for new "geopotential_temp.F90" CCPP scheme.
+
+List all existing files that have been modified, and describe the changes:
+
+held_suarez/held_suarez_1994.F90         - Code cleanup
+held_suarez/held_suarez_1994.meta        - Updated variable standard names and removed unused variables
+kessler/kessler.F90                      - Code cleanup and bug fixes
+kessler/kessler.meta                     - Updated variable standard names and removed unused variables
+kessler/kessler_update.F90               - Code cleanup
+kessler/kessler_update.meta              - Updated variable standard names and did some metadata cleanup
+suite_held_suarez_1994.xml               - Updated scheme names, and modified scheme list to better match CAM.
+suite_kessler.xml                        - Updated scheme names.
+utilities/physics_tendency_updaters.F90  - Updated variable names and dimensioning
+utilities/physics_tendency_updaters.meta - Updated variable standard names
+utilities/qneg.F90                       - Code cleanup
+utilities/qneg.meta                      - Updated variable standard names and removed unused variables
+utilities/state_converters.F90           - Code cleanup and updated variable/scheme names.
+utilities/state_converters.meta          - Updated variable standard names and scheme names,
+                                           and removed unused variables.
+utilities/static_energy.F90              - Code cleanup and updated variable dimensioning
+utilities/static_energy.meta             - Updated variable standard names
+
+List all standard names currently not in the CCPPStandardNames dictionary:
```

Comment:
I have added a new `NamesNotInDictionary.txt` file under the `doc` folder that contains the direct output from the stdname checking script.  I have also modified the script to output a timestamp so that in the future we can easily check whether or not the file is up-to-date.